main.js

```diff
@@ -0,0 +1,21 @@
+let elem = document.querySelector("li");
+    elem.remove();
+    // console.log("a");
+    // elem.style.display = "none";
+
+let el = document.querySelector("span");
+    el.remove();
+
+
+let lastPost = document.querySelector(".posts article");
+    lastPost.remove();
+
+
+let element = document.querySelectorAll(".featured h2, .posts h3")
+    for(let i = 0;i < element.length; i++) {
+        element[0].remove(), element[1].remove(), element[2].remove();        
+}
```

Comment:
```suggestion
let element = document.querySelectorAll(".featured h2, .posts h3")
    for(let i = 0;i < element.length; i++) {
        element[i].remove()       
}
```
You had the right idea using a loop to iterate over the h3 node list, but instead of statically only deleting the 0th, 1st, and 2nd elements, you can just use `element[i]` to make sure that you delete **every** element in the node list no matter how long it is.